docs/book/installation/requirements.rst

```diff
@@ -72,7 +72,7 @@ By default, the database connection is pre-configured to work with a following M
 Access rights
 -------------
 
-Most of the application folders and files require only read access, but a few folders need also the write access for the Apache user:
+Most of the application folders and files require only read access, but a few folders need also the write access for the Apache / Nginx user:
```

Comment:
Sorry for being annoying, but I think it should be `Apache/Nginx` - without spaces 😄

Reply:
No probem, I understand :)
I just pushed a new commit, but something weird happened: `docs/plugins/creating-plugin.rst` appeared to be modified on the pushed commit, but not on my local git repository... It looks up to date anyway.